tests/evergreen/test_api.py

```diff
@@ -162,13 +165,67 @@ def test_version_by_project(self, mocked_api):
         mocked_api.session.get.assert_called_with(url=expected_url, params=expected_params,
                                                   timeout=None)
 
+    def test_versions_by_project_time_window(self, mocked_api, sample_version, mocked_api_response):
+        version_list = [
+            deepcopy(sample_version),
+            deepcopy(sample_version),
+            deepcopy(sample_version),
+        ]
+        # Create a window of 1 day, and set the dates so that only the middle items should be
+        # returned.
+        one_day = timedelta(days=1)
+        one_hour = timedelta(hours=1)
+        start_date = parse_evergreen_datetime(version_list[1]['create_time'])
+        end_date = start_date - one_day
```

Comment:
Could we also rename `start_date` -> `before_date` and `end_date` -> `after_date`?

Reply:
yes, updated.